diff -urNp -x '*.orig' XML-SAX-1.02.org/Makefile.PL XML-SAX-1.02/Makefile.PL
--- XML-SAX-1.02.org/Makefile.PL	2019-06-14 01:33:42.000000000 +0200
+++ XML-SAX-1.02/Makefile.PL	2024-04-02 01:28:46.729057228 +0200
@@ -23,42 +23,3 @@ WriteMakefile(
     },
 );
 
-sub MY::install {
-    package MY;
-    my $script = shift->SUPER::install(@_);
-
-    # Only modify existing ParserDetails.ini if user agrees
-
-    my $write_ini_ok = 0;
-
-    eval { require XML::SAX };
-    if ($@) {
-        $write_ini_ok = 1;
-    }
-    else {
-        my $dir = File::Basename::dirname($INC{'XML/SAX.pm'});
-        if (-e File::Spec->catfile($dir, 'SAX', 'ParserDetails.ini')) {
-            $write_ini_ok =
-                ExtUtils::MakeMaker::prompt(
-                    "Do you want XML::SAX to alter ParserDetails.ini?", "Y"
-                ) =~ /^y/i;
-        }
-        else {
-            $write_ini_ok = 1;
-        }
-    }
-    
-    if ($write_ini_ok) {
-        $script =~ s/install :: (.*)$/install :: $1 install_sax_pureperl/m;
-        $script .= <<"INSTALL";
-
-install_sax_pureperl : pure_install
-\t\@\$(PERL) -MXML::SAX -e "XML::SAX->add_parser(q(XML::SAX::PurePerl))->save_parsers()"
-
-INSTALL
-
-    }
-
-    return $script;
-}
-
