--- XML-SAX-0.11/Makefile.PL.orig	Mon Jan 21 22:32:59 2002
+++ XML-SAX-0.11/Makefile.PL	Sat Sep 14 12:18:55 2002
@@ -9,20 +9,4 @@
     },    
 );
 
-sub MY::install {
-    package MY;
-    my $script = shift->SUPER::install(@_);
-    if (ExtUtils::MakeMaker::prompt("Do you want XML::SAX to alter ParserDetails.ini?", "Y") =~ /^y/i) {
-        $script =~ s/install :: (.*)$/install :: $1 install_sax_pureperl/m;
-        $script .= <<"INSTALL";
-
-install_sax_pureperl :
-\t\@\$(PERL) -MXML::SAX -e "XML::SAX->add_parser(q(XML::SAX::PurePerl))->save_parsers()"
-
-INSTALL
-
-    }
-
-    return $script;
-}
 
